pom.xml

```diff
@@ -43,6 +44,14 @@
 
     <dependencyManagement>
         <dependencies>
+            <!--  remove these 3 log4j once spring boot upgrade is available after 23/12/2021 -->
```

Comment:
You should no longer need this block

Reply:
Removed the comment but if I the dependecy block the dependecny reverts to 2.14.0 which is unsafe and will result in the security checks failing this is  in spite of having updated structured logging to 1.9.11

Reply:
Removed the block now but we are getting security complaint from concourse.